.travis.yml

```diff
@@ -1,4 +1,12 @@
 sudo: false
+env:
+  - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - g++-4.8
```

Comment:
I've added comments to the new microtime commit https://github.com/jedmao/cssnext/commit/dba072a1e332daea84535779abc9006bbc7732b2